client/components/Details.jsx

```diff
@@ -4,16 +4,16 @@ import Detail from './Detail.jsx';
 const Details = ({details, host}) => {
   const detailsToRender = [];
   if (details.typeOfPlace === 'entirePlace') {
-    detailsToRender.push(<Detail type={'entirePlace'} details={details} host={host}/>);
+    detailsToRender.push(<Detail key ={1} type={'entirePlace'} details={details} host={host}/>);
   }
   if (host.isSuperhost) {
-    detailsToRender.push(<Detail type={'superHost'} details={details} host={host}/>);
+    detailsToRender.push(<Detail key ={2} type={'superHost'} details={details} host={host}/>);
   } else if (details.houseRules.selfCheckIn.allowSelfCheckIn) {
-    detailsToRender.push(<Detail type={'selfCheckIn'} details={details} host={host}/>);
+    detailsToRender.push(<Detail key ={2} type={'selfCheckIn'} details={details} host={host}/>);
   }
-  detailsToRender.push(<Detail type={'enhancedClean'} details={details} host={host}/>);
-  detailsToRender.push(<Detail type={'cancellationPolicy'} details={details} host={host}/>);
-  detailsToRender.push(<Detail type={'houseRules'} details={details} host={host}/>);
+  detailsToRender.push(<Detail key ={3} type={'enhancedClean'} details={details} host={host}/>);
+  detailsToRender.push(<Detail key ={4} type={'cancellationPolicy'} details={details} host={host}/>);
+  detailsToRender.push(<Detail key ={5} type={'houseRules'} details={details} host={host}/>);
```

Comment:
could we pass down that _id mongo gives us instead of hardcoding the key? think it might just be extra complication for not much gain but everything looks good @maxmilpro

Reply:
good call out but i only have one _id to use. maybe i could enter type as the key instead of a different component.